component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java

```diff
@@ -192,6 +201,34 @@
                                 + "bcc='bcc1.account@gmail.com"
                                 + ")"
                                 + "define stream fooStream (name string, age int, country string);"),
+
+                @Example(description = "Following example illustrates how to publish events using the"
+                        + "email sink. Here files are also attached to the email."
```

Comment:
Need space between "the" and "email" :)

Reply:
fixed by 3d4e83b2f5d203578f81e8084b33d5770a6d0b18